pandas/tests/groupby/test_transform.py

```diff
@@ -743,14 +754,21 @@ def get_result(grp_obj):
                                   limit=limit)
 
     if test_series:
-        exp = pd.Series(exp_vals * 2)
-        exp.name = 'vals'
+        exp = exp.loc[:, 'A']
         grp = grp['vals']
         result = get_result(grp)
```

Comment:
Not that your change introduced it but can we get rid of the `get_result` function? Not sure why this was set up this way and adds unnecessary fluff

Reply:
Removed